src/components/Temp/Temp.js

```diff
@@ -0,0 +1,69 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import {View} from 'react-native';
+import useLocalize from '../../hooks/useLocalize';
+import useWindowDimensions from '../../hooks/useWindowDimensions';
+import Navigation from '../../libs/Navigation/Navigation';
+import styles from '../../styles/styles';
+import variables from '../../styles/variables';
+import BlockingView from '../BlockingViews/BlockingView';
+import * as Illustrations from '../Icon/Illustrations';
+
+const propTypes = {
+    /** Child elements */
+    children: PropTypes.node,
+
+    /** If true, child components are replaced with a blocking "not found" view */
+    shouldShow: PropTypes.bool,
+
+    /** The key in the translations file to use for the title */
+    titleKey: PropTypes.string,
```

Comment:
@hayata-suenaga content will be dynamic? If no we could remove translation keys from props

Reply:
@ArekChr I don't think content will be dynamic, so we can leave this.

Reply:
Let's simplify it then